hack/managed-kyma-migration/migrate.sh

```diff
@@ -25,8 +25,7 @@ fi
 istio_crs_count=$(kubectl get istios -n kyma-system --output json | jq '.items | length')
 
 if [ "$istio_crs_count" -gt 1 ]; then
-  echo "Multiple Istio CRs found, canceling migration"
-  exit 1
+  echo "WARNING: Multiple Istio CRs found"
```

Comment:
Why do we need to read the Istio CR count here? I think it'S not necessary to have it just for the echo, because there will also be an echo from

```shell
 elif [ "$STATUS" == "Error" ] && [ "$ISTIO_CR_COUNT" -gt 1 ]; then
    echo "More than one Istio CR present on the cluster. Script rename-to-default.sh might be required."
    exit 1
  fi
```